src/views/components-wallet/TheWalletSideMenu.vue

```diff
@@ -203,7 +203,18 @@ import ModuleSettings from '@/modules/settings/ModuleSettings';
 import ThemeSwitch from '@/components/theme-switch/ThemeSwitch';
 import { EventBus } from '@/core/plugins/eventBus';
 import { mapActions, mapGetters } from 'vuex';
-
+import { ETH, BSC, MATIC } from '@/utils/networks/types';
+const routeNames = {
```

Comment:
OMG YES, i dont know why but this made me happy

Reply:
we can probaly put this under core/router/helpers file? thens we can reuse it in more places and make another one fo the home page view

Reply:
yea I think routing can be its own refactor, lets do it later